src/app/js/main.js

```diff
@@ -180,38 +180,47 @@ function submit_slide(slide) {
   text = window.DATA.log.join('\n')
   element.style = 'font-size: 22px'
   element.innerHTML = '<a href="data:text/plain;charset=utf-8,' +
-    encodeURIComponent(text) + '" download="Cytoscape_Testing_results.txt">Download testing results</a>' +
-    '<br/> and <br/>' +
-    '<button type="submit" id="jiraBtn" onclick="submitReport()">Submit Jira Report</button>'
+    encodeURIComponent(text) + '" download="Cytoscape_Testing_results.txt">Download Test Results</a>' +
+    '<br/> <br/>' +
+    '<button type="submit" id="jiraBtn" class="btn btn-primary" onclick="sendData()">Submit Jira Report</button>'
 
   slide.appendChild(element)
 }
 
-// call our server rest api 
-function submitReport() {
+// Process report submission to the server
+function sendData() {
+  //TODO: update with new logger once available
+  var tester = document.getElementById('name').value
+  var testFeedback = document.getElementById('feedback').value
+  var testEnv = JSON.stringify(window.DATA['responses'].init.user_environment);
```

Comment:
Remember how this was stored in the Session? You can access from that object rather than window.data

Reply:
Please feel free to update, I am not confident these work. Don't won't to break a working code at this stage.